webapp/components/User/index.vue

```diff
@@ -47,7 +47,7 @@
             <b
               class="username"
               style="vertical-align: middle;"
-            >{{ user.name | truncate(trunc, 18) }}</b>
+            >{{ userName | truncate(18) }}</b>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:56:20Z" title="Thursday, May 2nd 2019, 7:56:20 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -47,7 +47,7 @@
             <b
               class="username"
               style="vertical-align: middle;"
-            >{{ user.name | truncate(trunc, 18) }}</b>
+            >{{ userName(user.name) | truncate(18) }}</b>
```


I would still be in favour of a `Username` component.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T11:26:31Z" title="Monday, May 6th 2019, 1:26:31 pm +02:00">May 6, 2019</time>_

---


Even tho this seems a nobel request it is not without implications. This would require alot of refactoring - no triage

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-06T20:47:11Z" title="Monday, May 6th 2019, 10:47:11 pm +02:00">May 6, 2019</time>_

---


Could you stop writing "no triage"? I think writing "I would not do X because of Y" does the job very well.